apps/web/screens/UniversitySearch/UniversitySearchDetails.tsx

```diff
@@ -130,72 +128,73 @@ const UniversityDetails: Screen<UniversityDetailsProps> = ({
     return dictionary
   }
 
-  const createTabContent = () => {
-    if (sortedCourses.length === 0) {
-      return
-    }
-    const tabList: Array<TabType> = []
-    const mappedDictionary = mapArrayToDictionary(
-      sortedCourses,
-      'semesterYearNumber',
-    )
-    let index = 0
-    for (const key in mappedDictionary) {
-      const value = mappedDictionary[key]
-      const mappedBySemester = mapArrayToDictionary(value, 'semesterSeason')
-
-      const contentItems: Array<React.ReactElement> = []
-      for (const x in mappedBySemester) {
-        contentItems.push(
-          <Box className={[styles.courseTypeIcon, styles.capitalizeText]}>
-            <Text variant="h4" color="blue400" paddingBottom={2} paddingTop={2}>
-              {n(x, TranslationDefaults[x])}
-            </Text>
-            {mappedBySemester[x].map((item) => {
-              return (
-                <Box
-                  display="flex"
-                  flexDirection="row"
-                  justifyContent="spaceBetween"
-                >
-                  <Text variant="h4" as="p" paddingBottom={1} paddingTop={1}>
-                    {locale === 'en' ? item.nameEn : item.nameIs}
-                  </Text>
-                  <Box className={styles.courseTypeIcon}>
-                    <Text
-                      fontWeight="semiBold"
-                      color={
-                        item.requirement === Requirement.MANDATORY
-                          ? 'red600'
-                          : item.requirement === Requirement.FREE_ELECTIVE
-                          ? 'blue600'
-                          : 'purple600'
-                      }
-                    >
-                      {item.requirement === Requirement.MANDATORY
-                        ? 'S'
-                        : item.requirement === Requirement.FREE_ELECTIVE
-                        ? 'V'
-                        : 'B'}
-                    </Text>
-                  </Box>
-                </Box>
-              )
-            })}
-          </Box>,
-        )
-      }
-
-      tabList.push({
-        id: index.toString(),
-        label: `${parseInt(key) + 1}. ${n('year', 'ár')}`,
-        content: contentItems,
-      })
-      index++
-    }
-
-    return tabList
-  }
+  // TODO THIS WILL BE ADDED BACK WHEN UNIVERSITIES RETURN THE COURSES FOR EACH PROGRAM
```

Comment:
Why delete the backend code and comment the frontend code, why not just delete both and rely on the diff when adding it back in?

Reply:
I was just trying to make sure this wouldnt get lost, the backend code is easy to add back and we have been working on multiple branches, which means things can get lost, so this was just me being careful about that. I can remove this also if it's better practice